__tests__/app.test.js

```diff
@@ -103,4 +95,43 @@ afterAll(() => db.end());
                 })
             })
         })
+    })
+
+    describe('/api/articles/:article_id/comments', () => {
+        it('should respond with an array of comments for a given article_id of which each comment should have the correct properties', () => {
+            return request(app)
+            .get('/api/articles/3/comments')
+            .expect(200)
+            .then(({body}) => {
+                const comments = body.comments
+                expect(comments.length).toBeGreaterThan(0)
+                expect(comments).toBeSortedBy('created_at', {descending: true})
+                comments.forEach((comment) => { 
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number)
```

Comment:
Solid happy path test and great use of `toMatchObject`, you could however be more specific about the `article_id` you're expecting all the comments to have 🔍

Reply:
resolved